src/test/java/io/opentracing/contrib/examples/multiple_callbacks/TestMultipleCallbacks.java

```diff
@@ -0,0 +1,44 @@
+package io.opentracing.contrib.examples.multiple_callbacks;
+
+import static com.jayway.awaitility.Awaitility.await;
+import static io.opentracing.contrib.examples.TestUtils.reportedSpansSize;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.junit.Assert.assertEquals;
+
+import io.opentracing.ActiveSpan;
+import io.opentracing.mock.MockSpan;
+import io.opentracing.mock.MockTracer;
+import io.opentracing.mock.MockTracer.Propagator;
+import io.opentracing.util.ThreadLocalActiveSpanSource;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import org.junit.Test;
+
+public class TestMultipleCallbacks {
+
+  private final MockTracer tracer = new MockTracer(new ThreadLocalActiveSpanSource(),
+      Propagator.TEXT_MAP);
+
+  @Test
+  public void test() throws Exception {
+    Client client = new Client(tracer);
+    try (ActiveSpan span = tracer.buildSpan("parent").startActive()) {
+      client.send("task1", span, 100);
+      client.send("task2", span, 200);
+      client.send("task3", span, 300);
+    }
```

Comment:
Yes, that's ok (just tested it). I was even going to make them sleep randomly, but I wanted the example to be clear about that (reversing their sleep time would still be fine as the user can still see such value).